packages/app-desktop/gui/MainScreen/MainScreen.tsx

```diff
@@ -160,6 +163,9 @@ class MainScreenComponent extends React.Component<Props, State> {
 			notePropertiesDialogOptions: {},
 			noteContentPropertiesDialogOptions: {},
 			shareNoteDialogOptions: {},
+			DefaultEditorDialogOptions: {
```

Comment:
```suggestion
			defaultEditorDialogOptions: {
```

**(formatting)** Prefer `camelCase` for object entries.